app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -63,6 +63,18 @@ class PatientRepository @Inject constructor(
     }
   }
 
+  @WorkerThread
+  fun search2(criteria: PatientSearchCriteria, facilityId: UUID): List<PatientSearchResult> {
+    return when (criteria) {
+      is Name -> TODO()
```

Comment:
FYI, We don't use Kotlin `TODO` class to define todo's since it will crash the app. Here we could've just returned the results from `searchByPhoneNumber2` directly instead of using the conditionals in this commit and define it in next commit where we will be using `searchByName2`.